draft-demarco-oauth-status-attestations.md

```diff
@@ -236,7 +236,7 @@ related to a specific Credential issued by the same Credential Issuer.
 +--------+----------+                         +----------+---------+
          |                                               |
          | HTTP POST /status                             |
-         |  credential_pop = $CredentialPoPJWT           |
+         |  credential_pop = [$CredentialPoPJWT]         |
```

Comment:
since the value is an array, wouldn't be better to rename this to `credentials_pop`?

@marinaado 
@fmarino-ipzs 
@OR13

Reply:
I wonder if these are all really "credentials"... or are they simply assertions related to a credential that had a confirmation claim?

- https://datatracker.ietf.org/doc/html/rfc7800#section-3.1
- https://datatracker.ietf.org/doc/html/rfc8747#section-3.1

Now that its an array, can each of these items demonstrate possession of a different key?

Can methods be mixed, for example, cred-1 uses COSE Key, cred-2 use JKT ?

Reply:
Given the updates from version 00 to 01, the term credentials is misleading because the items within that array are not credentials but assertions that prove credential possession. Therefore, we should rename this parameter to better reflect its purpose.

Reply:
another solution @SaraConsoliACN @OR13 to prevent the use of `application/json` would be this

`credential_pop="..."&credential_pop="..."&...`

taking into account that credential_pop is deprecable now and require a name change

Reply:
If the data posted is not credentials, it might be better to give it a more generic name, like `presentations` or `arguments`, or `parameters`

Reply:
we have a section defining the Digital Credential PoP here: https://github.com/peppelinux/draft-demarco-oauth-status-attestations/pull/36/files#diff-6018989306b8996e79d471eb123fc3b59796d1635384b6faac8cdb36cd5f49ccR297

I'm not sure about the name presentation

@SaraConsoliACN please revert the change to credential_pop to allow this PR to be merged and create another issue for discussing the json member name change

Reply:
@OR13 the discussion is moved to this issue: https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/43

I'd suggest to unblock this PR and continue the discussion about the nature of the parameters and the good semantic to have outside of this PR if we agree

@fmarino-ipzs @marinaado ^